kubernetes/resource_kubernetes_pod.go

```diff
@@ -41,7 +40,7 @@ func resourceKubernetesPod() *schema.Resource {
 				Required:    true,
 				MaxItems:    1,
 				Elem: &schema.Resource{
-					Schema: podSpecFields,
+					Schema: podSpecFields(false, false),
```

Comment:
I just moved this down here because that's how it's laid out in the Deployment resource, and I was comparing the two when investigating the [Pod volume bug](https://github.com/hashicorp/terraform-provider-kubernetes/issues/1085).